counsel.el

```diff
@@ -2695,6 +2715,15 @@ This uses `counsel-ag' with `counsel-ack-base-command' replacing
 Note: don't use single quotes for the regex."
   :type 'string)
 
+(defcustom counsel-rg-pcre-flag 'auto
+  "PCRE2 flag used by Ripgrep.
+If `auto', flag will be determined automatically on the next call
+to `counsel-rg'. If `nil', PCRE support is disabled unless
+\"--pcre2\" or \"-P\" is part of `counsel-rg-base-command'.
+
+Note: only Ripgrep version 0.10.0 and later support PCRE2."
+  :type '(choice string (const auto)))
```

Comment:
The docstring says "either `auto` or `nil`", but the `:type` says "either `auto` or a string".